test/timesheet.js

```diff
@@ -0,0 +1,223 @@
+            }
+            catch(error){
+                if(error.message !== 'estCrewSize is not valid') throw new Error('timesheet estCrewSize validation failed') ;
+            }
+
+        });
+
+        it('Timesheet Validation - estCrewHours', async function(){
+
+            let timesheet = new Timesheet(timesheetValues);
+            timesheet.estCrewHours = 'Not Valid Number';
+            try{
+                await timesheet.validate();
+            }
+            catch(error){
+                if(error.message !== 'estCrewHours is not valid') throw new Error('timesheet estCrewHours validation failed') ;
+            }
+
+        });
+
+        it('Timesheet Validation - tmCrewSize', async function(){
+
+            let timesheet = new Timesheet(timesheetValues);
+            timesheet.tmCrewSize = 'Not Valid Number';
+            try{
+                await timesheet.validate();
+            }
+            catch(error){
+                if(error.message !== 'tmCrewSize is not valid') throw new Error('timesheet tmCrewSize validation failed') ;
+            }
+
+        });
+
+        it('Timesheet Validation - tmCrewHours', async function(){
+
+            let timesheet = new Timesheet(timesheetValues);
+            timesheet.tmCrewHours = 'Not Valid Number';
+            try{
+                await timesheet.validate();
+            }
+            catch(error){
+                if(error.message !== 'tmCrewHours is not valid') throw new Error('timesheet tmCrewHours validation failed') ;
+            }
+
+        });
+
+        it('Timesheet Validation - lunchTaken', async function(){
+
+            let timesheet = new Timesheet(timesheetValues);
+            timesheet.lunchTaken = 'Not Valid Boolean';
+            try{
+                await timesheet.validate();
+            }
+            catch(error){
+                if(error.message !== 'lunchTaken is not valid') throw new Error('timesheet lunchTaken validation failed') ;
+            }
+
+        });
+
+        it('Timesheet Validation - jobFinished', async function(){
+
+            let timesheet = new Timesheet(timesheetValues);
+            timesheet.jobFinished = 'Not Valid Boolean';
+            try{
+                await timesheet.validate();
+            }
+            catch(error){
+                if(error.message !== 'jobFinished is not valid') throw new Error('timesheet jobFinished validation failed') ;
+            }
+
+        });
+
+        it('Timesheet Validation - offHauled', async function(){
+
+            let timesheet = new Timesheet(timesheetValues);
+            timesheet.offHauled = 'Not Valid Boolean';
+            try{
+                await timesheet.validate();
+            }
+            catch(error){
+                if(error.message !== 'offHauled is not valid') throw new Error('timesheet offhauled validation failed') ;
+            }
+
+        });
+
+        it('Timesheet Validation - yardsHauled', async function() {
```

Comment:
Test name says yardsHauled, but the test itself is testing offHauled.

Reply:
Fixed.